dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-initial-utilisation-and-fixed-fee.ts

```diff
@@ -0,0 +1,82 @@
+import { calculateInitialUtilisation, isValidDate } from '@ukef/dtfs2-common';
+import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
+import { calculateInitialFixedFee } from './calculate-initial-fixed-fee';
+
+export type RequiredParams = {
+  value?: number | null;
+  interestPercentage?: number | null;
+  dayCountBasis?: number | null;
+  coverStartDate?: string | Date | null;
+  coverEndDate?: string | Date | null;
+};
+
+/**
+ * parses date to the correct format
+ * date could be passed as string or different type of timestamp
+ * @param date
+ * @returns parsed date in correct format
+ */
+export const parseDate = (date: string | Date | number | null): Date => {
```

Comment:
Not a blocker for this PR. With this, we now have 3x `parseDate` functions and they are all different 🙃

One for another day.

Reply:
yes, have the same issue with isDate, need to all be moved to common another day